spec/requests/shopping_lists_spec.rb

```diff
@@ -41,7 +41,7 @@
 
         it 'returns the new list' do
           create_shopping_list
-          expect(response.body).to eq ShoppingList.last.to_json
```

Comment:
Turns out that this will be the master list that gets created in the after_create callback.